Source/FORMDataSource.h

```diff
@@ -65,6 +65,7 @@ typedef UICollectionViewCell * (^FORMFieldConfigureCellForItemAtIndexPath)(FORMF
 - (NSInteger)numberOfFields;
 - (NSArray *)forms;
 - (NSDictionary *)valuesDictionary;
+- (NSDictionary *)removedDynamicValues;
```

Comment:
I've been thinking that we should go for `values` and `removedValues`, like you don't say

"indexInteger"